questionpy_sdk/commands/package.py

```diff
@@ -43,12 +36,22 @@ def package(source: Path, manifest_path: Optional[Path], out_path: Optional[Path
     with manifest_path.open() as manifest_f:
         manifest = Manifest.parse_obj(yaml.safe_load(manifest_f))
 
+    if not out_path:
+        out_path = Path(f"{manifest.namespace}-{manifest.short_name}-{manifest.version}.qpy")
+    if out_path.exists():
+        if not click.confirm(f"The path '{out_path}' already exists. Do you want to override it?"):
+            click.echo("Aborting.")
+            return
```

Comment:
```suggestion
        if not click.confirm(f"The path '{out_path}' already exists. Do you want to overwrite it?"):
            click.echo("Aborting.")
            return
```

Ist mir vorher gar nicht aufgefallen